### PR TITLE
A benchmark for PlotMethod2

### PR DESCRIPTION
**Pull request by eggrobin**
It turns out that we mostly spend time in `lower_bound`:

profile

### CODE REVIEW

File: benchmarks/benchmarks.vcxproj [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
     <ClCompile Include="..\base\status.cpp" />
+    <ClCompile Include="..\ksp_plugin\planetarium.cpp" />
     <ClCompile Include="dynamic_frame.cpp" />
     <ClCompile Include="embedded_explicit_runge_kutta_nyström_integrator.cpp" />
     <ClCompile Include="ephemeris.cpp" />
     <ClCompile Include="hexadecimal.cpp" />
     <ClCompile Include="main.cpp" />
     <ClCompile Include="newhall.cpp" />
     <ClCompile Include="perspective.cpp" />
+    <ClCompile Include="planetarium_benchmarks.cpp" />
```
Comment (pleroy) [inline benchmarks/benchmarks.vcxproj]: In general we don't call benchmarks `_benchmarks`.

File: benchmarks/planetarium_benchmarks.cpp [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+            goes_8_orbit.StateVectors(goes_8_epoch));
+    auto goes_8_instance = ephemeris_->NewInstance(
+        {&goes_8_trajectory_},
+        Ephemeris<Barycentric>::NoIntrinsicAccelerations,
+        HistoryParameters());
+    CHECK_OK(ephemeris_->FlowWithFixedStep(goes_8_epoch + 100 * Day,
+                                           *goes_8_instance));
+  }
+
+  DiscreteTrajectory<Barycentric> const& GOES8Trajectory() const {
```
Comment (pleroy) [inline benchmarks/planetarium_benchmarks.cpp]: Should this function use snake_case?

File: benchmarks/planetarium_benchmarks.cpp [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+}  // namespace
+
+void RunBenchmark(benchmark::State& state,
+                  Perspective<Navigation, Camera> const& perspective) {
+  Satellites satellites;
+  Planetarium planetarium = satellites.MakePlanetarium(perspective);
+  RP2Lines<Length, Camera> lines;
+  int total_lines = 0;
+  int iterations = 0;
+  constexpr Instant now = "2000-01-21T04:41:30,5"_TT;
```
Comment (pleroy) [inline benchmarks/planetarium_benchmarks.cpp]: Where does this date come from?

File: benchmarks/planetarium_benchmarks.cpp [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+                 std::to_string(total_lines / iterations) + " lines within [" +
+                 DebugString(min_x) + ", " + DebugString(max_x) + "] × [" +
+                 DebugString(min_y) + ", " + DebugString(max_y) + "]");
+}
+
+void BM_PlanetariumPlotMethod2NearPolarPerspective(benchmark::State& state) {
+  RunBenchmark(state, PolarPerspective(near));
+}
+
+BENCHMARK(BM_PlanetariumPlotMethod2NearPolarPerspective);
```
Comment (pleroy) [inline benchmarks/planetarium_benchmarks.cpp]: Move all the macros together at the end.

File: benchmarks/planetarium_benchmarks.cpp (hunk 3 above)
Comment (eggrobin) [reply benchmarks/planetarium_benchmarks.cpp]: It's an eclipse that happened to be in January of 2000.

File: benchmarks/planetarium_benchmarks.cpp (hunk 2 above)
Comment (eggrobin) [reply benchmarks/planetarium_benchmarks.cpp]: Done.

File: benchmarks/benchmarks.vcxproj (hunk 1 above)
Comment (eggrobin) [reply benchmarks/benchmarks.vcxproj]: Yes, but if there are two translation units with the same name in the same project MSVC complains that the object files have the same name.

File: benchmarks/planetarium_benchmarks.cpp (hunk 4 above)
Comment (eggrobin) [reply benchmarks/planetarium_benchmarks.cpp]: Done.

**Pull request merged**